### PR TITLE
Introduces annotations for service object

### PR DESCRIPTION
**Pull request by gumdelli**
This PR allows for the service object to be annotated. Would be super useful for creating internal load balancers and such

### CODE REVIEW

**Comment by gumdelli** [comment]
@int128 Can you please take a look at this PR?

File: charts/kubernetes-dashboard-proxy/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 description: A reverse proxy with OpenID Connect for Kubernetes Dashboard
 name: kubernetes-dashboard-proxy
-version: 1.5.0
+version: 1.5.2
```
Comment (int128) [inline charts/kubernetes-dashboard-proxy/Chart.yaml]: Please bump the minor version as follows:

```suggestion
version: 1.6.0
```

**Review by int128 — COMMENTED** [review]
Thank you for your contribution and sorry for the late reply. Would you check my comment?

File: charts/kubernetes-dashboard-proxy/Chart.yaml (hunk 1 above)
Comment (gumdelli) [reply charts/kubernetes-dashboard-proxy/Chart.yaml]: @int128 done

**Comment by gumdelli** [comment]
@int128 updated as per your comment

**Pull request merged**